iac-extensions/terraform/src/main/java/org/sonar/iac/terraform/plugin/TerraformLanguage.java

```diff
@@ -25,7 +25,7 @@
 
 public class TerraformLanguage extends AbstractLanguage {
 
-  static final String KEY = "terraform";
+  public static final String KEY = "terraform";
```

Comment:
Nitpick : doesn't seems required to have this KEY public, maybe it was at some points ?